test/hotspot/jtreg/compiler/c2/irTests/TestVectorizationMismatchedAccess.java

```diff
@@ -150,7 +150,7 @@ static private void runAndVerify3(Runnable test, int offset) {
     }
 
     @Test
-    @IR(counts = { IRNode.LOAD_VECTOR_L, ">=1", IRNode.STORE_VECTOR, ">=1" })
+    @IR(applyIf = {"UseCompactObjectHeaders", "false"}, counts = { IRNode.LOAD_VECTOR_L, ">=1", IRNode.STORE_VECTOR, ">=1" })
```

Comment:
Putting `applyIf` and `counts` on different lines looks cleaner; like in the case way below.